addons/website_mail/models/email_template.py

```diff
@@ -1,19 +1,23 @@
 # -*- coding: utf-8 -*-
 
-from openerp.osv import osv
-from openerp.tools.translate import _
+from openerp import _
+from openerp import api, fields, models
 
 
-class EmailTemplate(osv.Model):
+class EmailTemplate(models.Model):
     _inherit = 'mail.template'
 
-    def action_edit_html(self, cr, uid, ids, context=None):
-        if not len(ids) == 1:
-            raise ValueError('One and only one ID allowed for this action')
-        url = '/website_mail/email_designer?model=mail.template&res_id=%d&enable_editor=1' % (ids[0],)
+    theme_xml_id = fields.Char('Theme XML ID',
```

Comment:
It stores the last selected theme used for designing that particular mail template. If user starts designing the same template again, last theme used for designing it will be loaded automatically instead of asking to select the one.